### PR TITLE
Add overview to subsite tabs

### PR DESCRIPTION
**Pull request by 95-martin-orion**

### CODE REVIEW

**Review by sergeisakov — APPROVED** [review]

File: docs/_book.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ upper_tabs:
     other:
     - name: "Guide & Tutorials"
       contents:
+      - title: "Overview"
```
Comment (lamberta) [inline docs/_book.yaml:13]: Can we name this "qsim and qsimh" to avoid a naming conflict with the "Overview" tab above?

Comment (95-martin-orion) [reply docs/_book.yaml:13]: We can, although I don't think we can change the path (`/qsim/overview`) to match. Is that OK?

Comment (lamberta) [reply docs/_book.yaml:13]: Yeah, that's fine. Can leave the filename

Comment (95-martin-orion) [reply docs/_book.yaml]: Done.

**Review by lamberta — APPROVED** [review]

**Pull request merged**